### PR TITLE
feat(): add tutorial for persistent user configs

### PR DESCRIPTION
**Pull request by Lordnoname**
add tutorial for persistent user configs

### CODE REVIEW

File: src/js/directives/mdCollapseMenu.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+angular.module('wgHilfe')
+  .directive('mdCollapaseMenu', function() {
```
Comment (DevVersion) [inline src/js/directives/mdCollapseMenu.js]: What's that?

Comment (Lordnoname) [inline src/js/directives/mdCollapseMenu.js]: wtf? That shouldn't be changed. I'll remove that

**Comment by DevVersion** [comment]
Can we not use this tutorial more generic?
 Like `Konfigurationen permanent speichern`, instead of only showing the `gitconfig`-case
This won't be a huge change, you just need to change the title of the tutorial and change a few messages, but you can keep using the `gitconfig` as a generic demonstration.

**Comment by DevVersion** [comment]
LGTM

File: src/content/guides/guides.json [hunk 2]
```diff
@@ -2,6 +2,10 @@
   "vlc": {
     "title": "VLC Medium abspielen",
     "path": "VLC/VLC.MD"
+  }
```
Comment (DevVersion) [inline src/content/guides/guides.json]: Missing comma

File: src/content/guides/guides.json [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+  "persistent_config": {
+    "title": "Permanente Konfigurationen",
```
Comment (DevVersion) [inline src/content/guides/guides.json]: :+1:

**Comment by Lordnoname** [comment]
Got a LGTM from @DevVersion

**Pull request merged**